docs/add-ons/configuration.md

```diff
@@ -57,6 +57,8 @@ then there will be a variable `TARGET` containing `beer` in the environment of y
 
 All add-ons are based on the latest Alpine Linux image. Home Assistant will automatically substitute the right base image based on the machine architecture. Add `tzdata` if you need to run in a different timezone. `tzdata` Is is already added to our base images.
 
+Images can be found [here](https://github.com/home-assistant/docker-base/blob/master/README.md). To test your add-on, you can build with these locally, e.g. on a 64-bit desktop `docker build --build-arg BUILD_FROM=ghcr.io/home-assistant/amd64-base`
```

Comment:
I think what you are adding is already covered by an existing page.

https://developers.home-assistant.io/docs/add-ons/testing#local-build

So it might be better to link to it, or at least make reference to it.